daemon/volumes.go

```diff
@@ -138,6 +139,13 @@ func (daemon *Daemon) registerMountPoints(container *container.Container, hostCo
 		}
 
 		if label.RelabelNeeded(bind.Mode) {
+			if err := os.MkdirAll(bind.Source, 0755); err != nil {
```

Comment:
Os was already imported, system was not and after reading system, it looked like it just called os, but I have no problem switching to it, if you want.  What does system do that os does not?